src/scope.ml

```diff
@@ -257,7 +257,8 @@ let scope : mode -> sig_state -> env -> p_term -> tbox = fun md ss env t ->
         fatal t.pos "[%a] is not allowed in a LHS." Print.pp Type
     | (P_Type          , _                ) -> _Type
     | (P_Iden(qid,_)   , M_LHS(_,p)       ) -> find_qid true p ss env qid
-    | (P_Iden(qid,_)   , M_Term(_,Private)) -> find_qid false true ss env qid
+    | (P_Iden(qid,_)   , M_Term(_,Privat )) ->
+        find_qid false true ss env qid
```

Comment:
No need for a separate line here.

Reply:
Fixed in d7b99a6f1252f9d51d90833cc70c9ff9774a01b8.